Objective-C/Tests/ReplicatorTest+Main.m

```diff
@@ -1763,6 +1772,76 @@ - (void) testPushAndPullExpiredDocument_SG {
     AssertNil([savedDoc stringForKey: propertyKey]);
 }
 
+#pragma mark - SG (TLS with chain of certs)
+/**
+ - Use the `certs.pem` & `certs.key` in the `LiteCore/Replicator/tests/data/ssl_config.json`
+    - "SSLCert": "../../../../../Objective-C/Tests/Support/chain-certs/certs.pem",
+    - "SSLKey":  "../../../../../Objective-C/Tests/Support/chain-certs/certs.key",
+ */
+// Replicator with non-matching cert + sg with muliple certs
+- (void) testMultipleCertSGWithNonMatchingCert_SG {
+    id target = [self remoteEndpointWithName: @"itunes" secure: YES];
+    if (!target)
+        return;
+    
+    NSURL* url = [NSURL URLWithString: @"wss://localhost:4986/itunes"];
+    target = [[CBLURLEndpoint alloc] initWithURL: url];
+    
+    self.disableDefaultServerCertPinning = YES;    // without this, SSL handshake will fail
+    id config = [self configWithTarget: target type: kCBLReplicatorTypePull continuous: NO];
+    [self run: config errorCode: CBLErrorTLSCertUnknownRoot errorDomain: CBLErrorDomain];
+}
+
+- (SecCertificateRef) certChains: (NSString*)name {
```

Comment:
This method is getting only one cert and cert chain. Probably change to `getCert:`?

Reply:
Alternatively, you can try something like this. If it works, you just use the PEM string in the test without creating the DER file. I did this in LiteCore test.

```
NSString* pem = "<CERT IN PEM format>"
NSData *certData = [[NSData alloc] initWithBase64Encoding: pem];
SecCertificateRef parsedCertificate = SecCertificateRef cert = SecCertificateCreateWithData(NULL, (__bridge CFDataRef)certData);
```